components/post_view/post_list/post_list.jsx

```diff
@@ -243,7 +250,9 @@ export default class PostList extends React.PureComponent {
         if (this.extraPagesLoaded > MAX_EXTRA_PAGES_LOADED) {
             // Prevent this from loading a lot of pages in a channel with only hidden messages
             // Enable load more messages manual link
-            this.setState({autoRetryEnable: false});
+            if (this.state.autoRetryEnable) {
```

Comment:
I don't think this should be needed since the PureComponent should prevent the component from re-rendering if autoRetryEnable was already false